webpack.config.js

```diff
@@ -0,0 +1,209 @@
+              options: {
+                plugins: function (loader) {
+                  return [ autoprefixer({ browsers: [ '> 1%' ] }) ]
+                }
+              }}, 'sass-loader']
+        }) : [
+          'style-loader',
+          {
+            loader: 'css-loader',
+            options: {
+              importLoaders: 2,
+              modules: true,
+              sourceMap: true,
+              localIdentName: '[name]__[local]___[hash:base64:5]'
+            }
+          },
+          {
+            loader: 'postcss-loader',
+            options: {
+              plugins: function (loader) {
+                return [ autoprefixer({ browsers: [ '> 1%' ] }) ]
+              }
+            }
+          },
+          'sass-loader'
+        ]
+      },
+      {
+        test: /\.css$/,
+        use: isProduction ? ExtractTextPlugin.extract({
+          fallback: 'style-loader',
+          use: [{
+            loader: 'css-loader',
+            options: {
+              importLoaders: 1
+            }
+          }, {
+            loader: 'postcss-loader',
+            options: {
+              plugins: function (loader) {
+                return [ autoprefixer({ browsers: [ '> 1%' ] }) ]
+              }
+            }
+          }]
+        }) : [
+          'style-loader',
+          'css-loader',
+          {
+            loader: 'postcss-loader',
+            options: {
+              plugins: function (loader) {
+                return [ autoprefixer({ browsers: [ '> 1%' ] }) ]
+              }
+            }
+          }
+        ]
+      },
+      {
+        test: /\.svg(\?v=\d+\.\d+\.\d+)?$/,
+        use: [{
+          loader: 'url-loader',
+          options: {
+            limit: 10000,
+            mimetype: 'image/svg+xml'
+          }
+        }]
+      }
+    ]
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
+        BROWSER: true,
+        NODE_ENV: isProduction ? '"production"' : '"development"'
+      },
+      __CLIENT__: true,
+      __DEVELOPMENT__: !isProduction,
+      __DEVTOOLS__: true,  // <-------- DISABLE redux-devtools HERE
+      __SERVER__: false
+    })
+  ]
+}
+
+
+if (isProduction) {
+  webpackConfig.plugins.push(
+    new BundleListPlugin(),
+    // css files from the extract-text-plugin loader
+    new ExtractTextPlugin({
+      // write css filenames into webpack-assets.json
+      filename: function(getPath) {
+        const filename = getPath('[chunkhash].[name].css')
+        let assets
+        try {
+          assets = require('./webpack-assets.json')
+        } catch (e) {
+          assets = defaulAssets
+        }
+        assets.stylesheets.push(webpackConfig.output.publicPath + filename)
+        fs.writeFileSync('./webpack-assets.json', JSON.stringify(assets))
+        return filename
+      },
+      disable: false,
+      allChunks: true
+    }),
+    new webpack.optimize.UglifyJsPlugin({
+      compress: {
+        warnings: false
```

Comment:
the default value of `compress.warning` is `false` in webpack@^2
https://webpack.js.org/guides/migrating/#uglifyjsplugin-warnings